Lib/distutils/tests/test_check.py

```diff
@@ -13,6 +14,10 @@
     pygments = None
 
 
+HERE = Path(__file__).parent
+INCLUDE_TEST_PATH = (HERE / 'includetest.rst').relative_to(Path.cwd())
```

Comment:
Could you use old-school os.path functions to ease backport?

Reply:
to what will this be backported that doesn’t have pathlib?

Reply:
OK, done!